src/views/organisations/overview.html

```diff
@@ -82,7 +94,9 @@ <h2 class="govuk-heading-m">
               {% endif %}
             </h2>
             <div class="govuk-task-list__hint">
-              {% if dataset.status == 'Not submitted' %}
+              {% if dataset.notice %}
+                {{ noticeHeading(dataset, dataset.notice) }}
```

Comment:
Content here should be `You must update this dataset by 31 December 2024` without the dataset name